docker/README.md

```diff
@@ -10,13 +10,13 @@ container with the PDO contracts. For the most part, this directory
 could be copied to any host (even without PDO otherwise installed) to
 build, configure, and execute PDO contracts code.  Also note that the
 makefile assumes you have already built all containers from the
-`private-data-objects` repository.  If not, just run `make repository;
-make -C repository/private-data-objects/docker`.  For clarity the
-scripts in the directory are intended to serve two purposes. The first
-is to provide the basic instructions for building images for running
-the various contract families in the PDO contracts repository. The
-images may be used for a variety of uses including demonstration,
-testing and interacting with PDO contracts.
+`private-data-objects` repository.
```

Comment:
If folks read this README, they might execute the targets directly in this makefile in which case the top-level pdo_docker target is not executed? Wouldn't it be better to move the `pdo_docker` target from the top-level to this makefile or at least reference here the top-level target?

Reply:
i think the readme is (intentionally) agnostic to HOW the images are created. happy to have the discussion about where the targets should reside. moving it down makes some sense... leaving it at the top level makes it easier to access.

Reply:
Moved down to the docker directory. I think you're right that it makes more sense there.